kubernetes/resource_kubernetes_persistent_volume.go

```diff
@@ -11,6 +11,7 @@ import (
 	api "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
There's another import of `k8s.io/apimachinery/pkg/apis/meta/v1` right above this. Can we use only one?

Reply:
Indeed, fixed.